internal/terraform/node_resource_validate.go

```diff
@@ -217,6 +217,26 @@ var connectionBlockSupersetSchema = &configschema.Block{
 			Type:     cty.String,
 			Optional: true,
 		},
+		"proxy_scheme": {
+			Type:     cty.String,
+			Optional: true,
+		},
+		"proxy_host": {
+			Type:     cty.String,
+			Optional: true,
+		},
+		"proxy_port": {
+			Type:     cty.String,
```

Comment:
Same note as above—I think this should be a `cty.Number`.

Reply:
Fixed